docs/content/preview/reference/configuration/yb-tserver.md

```diff
@@ -1952,6 +1832,115 @@ Starting from version 2.18, the default is `-1`. Previously it was `4`.
+
+##### --auto_compact_check_interval_sec
+
+{{% tags/wrap %}}
+
+
+Default: `60`
+{{% /tags/wrap %}}
+
+The interval at which the full compaction task will check for tablets eligible for compaction (both for the statistics-based full compaction and scheduled full compaction features). `0` indicates that the statistics-based full compactions feature is disabled.
+
+##### --auto_compact_stat_window_seconds
+
+{{% tags/wrap %}}
+
+
+Default: `300`
+{{% /tags/wrap %}}
+
+Window of time in seconds over which DocDB read statistics are analyzed for the purpose of triggering full compactions to improve read performance. Both `auto_compact_percent_obsolete` and `auto_compact_min_obsolete_keys_found` are evaluated over this period of time.
+
+`auto_compact_stat_window_seconds` must be evaluated as a multiple of `auto_compact_check_interval_sec`, and will be rounded up to meet this constraint. For example, if `auto_compact_stat_window_seconds` is set to `100` and `auto_compact_check_interval_sec` is set to `60`, it will be rounded up to `120` at runtime.
+
+##### --auto_compact_percent_obsolete
+
+{{% tags/wrap %}}
+
+
+Default: `99`
+{{% /tags/wrap %}}
+
+The percentage of obsolete keys (over total keys) read over the `auto_compact_stat_window_seconds` window of time required to trigger an automatic full compaction on a tablet. Only keys that are past their history retention (and thus can be garbage collected) are counted towards this threshold.
+
+For example, if the flag is set to `99` and 100000 keys are read over that window of time, and 99900 of those are obsolete and past their history retention, a full compaction will be triggered (subject to other conditions).
+
+##### --auto_compact_min_obsolete_keys_found
+
+{{% tags/wrap %}}
+
+
+Default: `10000`
+{{% /tags/wrap %}}
+
+Minimum number of keys that must be read over the last `auto_compact_stat_window_seconds` to trigger a statistics-based full compaction.
+
+##### --auto_compact_min_wait_between_seconds
+
+{{% tags/wrap %}}
+
+
+Default: `0`
+{{% /tags/wrap %}}
+
+Minimum wait time between statistics-based and scheduled full compactions. To be used if statistics-based compactions are triggering too frequently.
+
+##### --scheduled_full_compaction_frequency_hours
+
+{{% tags/wrap %}}
+
+
+Default: `0`
+{{% /tags/wrap %}}
+
+The frequency with which full compactions should be scheduled on tablets. `0` indicates that the feature is disabled. Recommended value: `720` hours or greater (that is, 30 days).
+
+##### --scheduled_full_compaction_jitter_factor_percentage
+
+{{% tags/wrap %}}
+
+
+Default: `33`
+{{% /tags/wrap %}}
+
+Percentage of `scheduled_full_compaction_frequency_hours` to be used as jitter when determining full compaction schedule per tablet. Must be a value between `0` and `100`. Jitter is introduced to prevent many tablets from being scheduled for full compactions at the same time.
```

Comment:
```suggestion
Percentage of [scheduled_full_compaction_frequency_hours](#scheduled-full-compaction-frequency-hours) to be used as jitter when determining full compaction schedule per tablet. Must be a value between `0` and `100`. Jitter is introduced to prevent many tablets from being scheduled for full compactions at the same time.
```